tools/docker/riak/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM hectcastro/riak
```

Comment:
Unfortunately not, the only available tag for this image is `latest`...

The Riak image is used only to test MongooseIM with Riak. The infrastructure enabling docker doesn't have Riak as a regular service, as it was before, that's why Riak is started in container.